addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -331,7 +331,7 @@ var SnippetEditor = Widget.extend({
      * Makes the editor overlay cover the associated snippet.
      */
     cover: function () {
-        if (!this.isShown() || !this.$target.length) {
+        if (!this.isShown() || !this.$target.length || !this.$target[0].ownerDocument.defaultView) {
```

Comment:
This is either an unrelated bugfix that was squashed with the rest, or a special case that is possible now which was not possible before with the old widgets. If the latter, I'd like to know why. Can't we avoid it ?